website/source/docs/builders/vagrant.html.md

```diff
@@ -74,11 +74,21 @@ one, by using `global_id` instead of `source_box`.
     not recommended since OVA files can be very large and corruption does happen
     from time to time.
 
--   `vagrantfile_template` (string) - a path to a golang template for a
+-   `template` (string) - a path to a golang template for a
     vagrantfile. Our default template can be found
     [here](https://github.com/hashicorp/packer/blob/a6d1d852bb0643e3593cc5d3dfe5ed80c4409b65/builder/vagrant/step_create_vagrantfile.go#L23-L30). So far the only template variables available to you are {{ .BoxName }} and
     {{ .SyncedFolder }}, which correspond to the Packer options `box_name` and
-    `synced_folder`
+    `synced_folder`.
+
+    You must provide a template if your default vagrant provider is Hyper-V.
+    Below is a Hyper-V compatible template.
```

Comment:
```suggestion
    Bellow is a Hyper-V compatible template.
```

Reply:
Below is actually spelled correctly here.

https://www.merriam-webster.com/dictionary/below